modules/projects/server/controllers/project.load.controller.js

```diff
@@ -88,6 +88,32 @@ module.exports = function(file, req, res, opts) {
                 .saveDocument(project)
                 .then(rs);
             });
+        } else if (finalPhaseCode === "closed") {
```

Comment:
This only had its phases bumped since it is not the 5th element, and not the 4th, in the array.